### PR TITLE
release: Include zephyr controller qualification fixes and release documentation updates

### PR DESCRIPTION
**Pull request by joerchan**
Update manifest entry for zephyr to include controller fixes
needed for qualification.

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

### CODE REVIEW

**Review by ru-fu — CHANGES_REQUESTED** [review]
I'll add a commit.

File: doc/release-notes-1.3.2.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+* Zephyr Bluetooth LE controller:
+
+  * Fixed invalid disconnect reason in the disconnect complete event.
+  * Changed the controller HCI version to Bluetooth Core Specification v5.1.
```
Comment (ru-fu) [inline doc/release-notes-1.3.2.rst:53]: This sounds confusing. Do you mean "Changed the controller HCI version to be compatible with/to follow the Bluetooth Core Specification v5.1."?
(Please don't change right now, I'm working on the branch ;) )

Comment (joerchan) [reply doc/release-notes-1.3.2.rst:53]: I only changed the version number, and the version number means Bluetooth Core Specification v5.1

Comment (ru-fu) [reply doc/release-notes-1.3.2.rst:53]: Changed from where? This wasn't in the previous release notes.

I don't understand what we're trying to say here. How can the HCI version be the Bluetooth specification?
But if that's what we want to say, fine.

**Review by ru-fu — COMMENTED** [review]
I updated the versions now. Please check, and squash the commit with release notes fixes.

File: doc/release-notes-1.3.2.rst (hunk 1 above)
Comment (joerchan) [reply doc/release-notes-1.3.2.rst:53]: https://www.bluetooth.com/specifications/assigned-numbers/link-layer/

In 1.3.0 and 1.3.1 it is 5.2. We also didn't include qualification before, so we haven't mentioned it's version either.

Comment (ru-fu) [reply doc/release-notes-1.3.2.rst:53]: I think it should be something like 
```suggestion
  * Selected the controller HCI version that corresponds to the Bluetooth Core Specification v5.1.
```
But if that's wrong, leave as is.

File: doc/nrf/conf.py [hunk 2]
```diff
@@ -110,9 +110,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.3.1'
+version = '1.3.2'
```
Comment (carlescufi) [inline doc/nrf/conf.py:113]: @tejlmand we need to read this from the `VERSION` file

File: doc/release-notes-1.3.2.rst (hunk 1 above)
Comment (carlescufi) [reply doc/release-notes-1.3.2.rst:53]: This should be: "The controller HCI version, reported with the Read Local Version Information HCI command, has been changed to match version 5.1 of the Bluetooth Core Specification".

Comment (ru-fu) [reply doc/release-notes-1.3.2.rst:53]:
```suggestion
  * Changed the controller HCI version (reported with the Read Local Version Information HCI command) to match version 5.1 of the Bluetooth Core Specification.
```

File: doc/nrf/conf.py (hunk 2 above)
Comment (carlescufi) [reply doc/nrf/conf.py:113]: (not in this PR)

**Review by carlescufi — APPROVED** [review]
Checked SHA

**Pull request merged**